### PR TITLE
Clean up old parquet stuff

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

**Comment by tpoterba** [comment]
should make the tests significantly faster by removing the slowest one.

File: src/main/scala/is/hail/utils/richUtils/RichSQLContext.scala [hunk 1]
```diff
@@ -1,29 +0,0 @@
-package is.hail.utils.richUtils
-
-import is.hail.utils._
-import org.apache.spark.rdd.RDD
-import org.apache.spark.sql.functions._
-import org.apache.spark.sql.{Row, SQLContext}
-
-class RichSQLContext(val sqlContext: SQLContext) extends AnyVal {
-  def readParquetSorted(dirname: String, selection: Option[Array[String]] = None): RDD[Row] = {
```
Comment (cseed) [inline src/main/scala/is/hail/utils/richUtils/RichSQLContext.scala]: This makes me so happy.

File: src/main/scala/is/hail/utils/richUtils/RichSQLContext.scala [hunk 2]
```diff
@@ -1,29 +0,0 @@
...
-
-    var df = sqlContext.read.parquet(dirname + "/part-*")
-    selection.foreach { cols =>
-      df = df.select(cols.map(col): _*)
-    }
-
-    val rdd = df.rdd
-
-    val oldIndices = rdd.partitions
-      .map { p => getParquetPartNumber(partitionPath(p)) }
```
Comment (cseed) [inline src/main/scala/is/hail/utils/richUtils/RichSQLContext.scala]: I wish we could get rid of `partitionPath`, too, but it is still used by `textFilesLines`.

File: src/main/scala/is/hail/utils/richUtils/RichSQLContext.scala [hunk 3]
```diff
@@ -1,29 +0,0 @@
-package is.hail.utils.richUtils
-
-import is.hail.utils._
-import org.apache.spark.rdd.RDD
-import org.apache.spark.sql.functions._
-import org.apache.spark.sql.{Row, SQLContext}
-
-class RichSQLContext(val sqlContext: SQLContext) extends AnyVal {
```
Comment (cseed) [inline src/main/scala/is/hail/utils/richUtils/RichSQLContext.scala]: You need to nuke the corresponding implicit.

**Pull request merged**